packages/lodestar/src/network/gossip/interface.ts

```diff
@@ -22,14 +22,21 @@ import {IBeaconChain} from "../../chain";
 import {ForkDigest} from "@chainsafe/lodestar-types";
 
 export interface IGossipEvents {
+  [attSubnetTopic: string]: (attestationSubnet: {attestation: Attestation; subnet: number}) => void;
+  // @ts-ignore
   [GossipEvent.BLOCK]: (signedBlock: SignedBeaconBlock) => void;
-  [GossipEvent.ATTESTATION_SUBNET]: (attestationSubnet: {attestation: Attestation; subnet: number}) => void;
+  // @ts-ignore
   [GossipEvent.ATTESTATION]: (attestation: Attestation) => void;
+  // @ts-ignore
   [GossipEvent.AGGREGATE_AND_PROOF]: (attestation: SignedAggregateAndProof) => void;
+  // @ts-ignore
   [GossipEvent.VOLUNTARY_EXIT]: (voluntaryExit: SignedVoluntaryExit) => void;
+  // @ts-ignore
   [GossipEvent.PROPOSER_SLASHING]: (proposerSlashing: ProposerSlashing) => void;
+  // @ts-ignore
```

Comment:
Oh sorry, I should been more clearer, don't declare dynamic event here, just add @ts-ignore on two places we are using it